content/rule_book/roles_and_responsibilities.md

```diff
@@ -2,68 +2,96 @@
+
+##### Reference Questions
+
+On a Reference type question, if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will ask, "What is your question?"
+
+##### Answer Correction
+
+On a Quote/Finish type question, once the quizzer has quoted to the end of the required verse or verses, if the quote is not yet word-perfect, the quizmaster will say, "Again." This will occur as often as needed until the quizzer quotes the verse perfectly or the question answer duration period expires. The quizmaster is not allowed to say "again" until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
+
+For all other question types, once all of the information in both the question and answer has been given, but something needs to be fixed, the quizmaster will say "Again".
+
+##### Quote is Complete
+
+On a Situation type question, when the quizzer has completed the quotation, the quizmaster will say, "Quote is complete."
+
+##### Appending an Answer
+
+On an INT or MA Question, if all the information in the answer text has been given but the quizzer is missing information in the question text, the quizmaster will say "More".
+
+On a Situation question, because the quotation is part of the question, the quizmaster will say "More" when required information from the quotation is missing.
+
+On a Quote or Reference question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say "More".
 
 #### Commentary
-On a CVR/MACVR, in the scenario where the entire Reference (which is part of the "Question") was not read by the quizmaster, the quizmaster has two prompts to give, and must give them in appropriate order and with appropriate timing. If the quizzer provides all information in the question "text" and the answer "text", because the quizzer has not yet finished saying the complete Reference (part of the "question"), the quizmaster must prompt with "More." The quizmaster cannot prompt with "What is your question?" yet, because the quizzer has not provided all information in the "question" (which includes the Reference) and the answer.
+
+On a CVR/MACVR, in the scenario where the entire reference (which is part of the question text) was not read by the quizmaster, the quizmaster has 2 prompts to give, and must give them in appropriate order and with appropriate timing. If the quizzer provides all information in the question text and the answer text, because the quizzer has not yet finished saying the complete Reference, which is part of the question, the quizmaster must prompt with "More." The quizmaster cannot prompt with "What is your question?" yet, because the quizzer has not provided all information in the question (which includes the reference) and the answer.
```

Comment:
Massive subtlety that I think could be a point of interest for those reviewing the rewrite:
I don't think QMs today do this the RB prescribes. But that's largely because we JUST codified "question" to include the reference, which is something I don't think people do now. Thus, they prompt for "what is your question", then say "more" if the quizzer needs to finish the reference

Reply:
This is consistent with IBQ and at least how we've been QMing in PNW this season and last. Not sure about other districts, though.

Reply:
I've definitely witnessed QMs prompt for question, then upon acceptable question, say "more" so the quizzer can complete the reference.

Reply:
Yikes. That's gotta be confusing. At IBQ or PNW? Do you remember about when, roughly?

Reply:
I don't remember. But I don't feel like it's rare. Maybe GWI a few years back?

On QTs, where the Reference wasn't completed, QMs routinely say "again", then once the quizzer has the verse correct, will either stay silent, or say "more"

I REALLY don't think it's rare for QMs to not consider the reference part of the question. So will do all their other prompts first. Then "more"

Reply:
I guess the RB allows "again" on QTs though, since it just says "when the quizzer reaches the end of the verse"